src/System.Text.Json/src/System/Text/Json/Document/JsonDocument.TryGetProperty.cs

```diff
@@ -9,6 +9,9 @@ namespace System.Text.Json
 {
     public sealed partial class JsonDocument
     {
+        /// <summary>
+        ///   This is an implementation detail and MUST NOT be called by source-package consumers.
```

Comment:
📝 These comments are not helpful. It would be better to replace them with an attribute that can eventually be supported by a diagnostic analyzer. The comments themselves should be written from the perspective of the caller, which in this case will never be source-package consumers.